meta-balena-common/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -53,6 +53,10 @@ do_install_append() {
     # remove these empty not-used (at this moment) directories so we don't have to package them
     rmdir ${D}${libdir}/NetworkManager/conf.d
     rmdir ${D}${libdir}/NetworkManager/VPN
+
+    if [ "${@bb.utils.filter('PACKAGECONFIG', 'systemd', d)}" ]; then
+        sed -i "/ExecStart=/i ExecStartPre=/bin/sh -c 'find /var/lib/NetworkManager -type f -print0 | grep -E \"timestamps|seen-bssids.[a-zA-Z0-9]{6}\" | xargs rm -f'" ${D}/lib/systemd/system/NetworkManager.service
```

Comment:
We already have a drop-in for this service. It'll be easier to add `ExecStartPre=` there rather than this sed.

Also, that one-liner is too long. I'd put it in a separate script and call that from the service.

Reply:
Fixed below.